src/app/providers/QueryClientProvider.tsx

```diff
@@ -1,6 +1,14 @@
 import { QueryClient, QueryClientProvider as MyQueryClientProvider} from '@tanstack/react-query';
 
-const queryClient = new QueryClient();
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      staleTime: 5 * 60 * 1000, 
+      gcTime: 10 * 60 * 1000,
```

Comment:
시간표현 깔끔하네요 ㅎㅎ 소소하게 numeric separator를 이용하면 큰 숫자를 조금 더 보기 좋게 표시할 수 있으니 참고하시면 좋겠네요
```suggestion
      staleTime: 5 * 60 * 1_000, 
      gcTime: 10 * 60 * 1_000,
```

Reply:
오 꿀팁이군요, 감사합니다.